src/vulkan.cpp

```diff
@@ -2079,8 +2079,17 @@ static VkResult overlay_CreateInstance(
                              &instance_data->vtable);
    instance_data_map_physical_devices(instance_data, true);
 
+   parse_overlay_config(&instance_data->params, getenv("MANGOHUD_CONFIG"));
+
+   //check for blacklist item in the config file
+   std::stringstream ss(instance_data->params.blacklist);
+   std::string token;
+   while (std::getline(ss, token, ',')) {
```

Comment:
Then probably make this `parse_str_tokenized` (or something, i can't name things :D ) parsed into a vector, for other APIs.

Reply:
Ok now we have parse_str_tokenize .. I am not any better :)